_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -20,19 +20,19 @@ acknowledgments:
 
 ## Applicability
 
-The rule applies to any [keyboard event][] for which all is true: 
+The rule applies to any [keyboard event][] for which all of the following is true: 
 - the event's attribute `key` is a [printable character][] key; and
-- the event's method `getModifierState` returns `false` for any of the [valid modifier keys][].
+- the event's method `getModifierState` returns `false` for any of the [valid modifier keys][]; and
```

Comment:
```suggestion
- the event's method `getModifierState` returns `false` for each of the [valid modifier keys][]; and
```
I think "any" would mean that one of them is enough, which is clearly not what we want. Might need help from @EmmaJP or other native speaker here.